### PR TITLE
[Task] errorpage: post not found & styling about heading

### PR DESCRIPTION
**Pull request by ranjabi**
## Story/Task
_errorpage: post not found & styling about heading_

## Summary
-   _using post not found message for error url post page_
-   _styling about heading_
-   _using yarn for husky pre-commit_
-   _change lint command_
In my case husky pre commit is failed when using npx. The previous lint command doesn't lint anything because the path is not specified.

## Test Plan
_none_

## Screenshot
image

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx lint-staged
```
Comment (IloveNooodles) [inline .husky/pre-commit:4]: ini udah di run dan jalan?

File: src/components/Rumpun.tsx [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
 
-const Rumpun: React.FC<RumpunProps> = ({ id, name }) => {
-    return (
-        <div className="flex flex-col gap-4 justify-center items-center">
-            <div className="bg-LightBrown w-full aspect-square hover:scale-105 transition-transform"></div>
-            <h2 className="text-3xl">{name}</h2>
-        </div>
-    );
-};
+const Rumpun: React.FC<RumpunProps> = ({ name }: { name: string }) => (
```
Comment (IloveNooodles) [inline src/components/Rumpun.tsx:3]: kalo udah make props gausah di infer lagi typesnya biarin aja udah auto

**Review by IloveNooodles — APPROVED** [review]
approved with some comments

File: .husky/pre-commit (hunk 1 above)
Comment (ranjabi) [reply .husky/pre-commit:4]: udah

**Pull request merged**